tests/lib/Limiters/UserPassBaseLimiterTest.php

```diff
@@ -23,7 +23,7 @@ public function testKeyVariesWithInput(): void
         $this->assertNotEquals($key1, $key2, 'keys should vary');
     }
 }
-
+// phpcs:ignore
```

Comment:
Curious:  what does this do? I don't see anything that would trigger an issue

Reply:
The test class has a second class (that is used in the test) defined in the file, so this ignore is just saying 'it is okay to have the second class'